server/app/models/external_volume.rb

```diff
@@ -0,0 +1,8 @@
+class ExternalVolume
+  include Mongoid::Document
+  include Mongoid::Timestamps
+
+  field :name, type: String
+  
+  has_one :volume
```

Comment:
Does this work like you expect it to? Per the [Mongoid relations doc](https://mongoid.github.io/old/en/mongoid/docs/relations.html#has_one), this means that each volume in the volumes` collection will have a `external_volume_id`. This means that each `Volume` can only be referenced by one `ExternalVolume`?